depthai_examples/src/stereo_inertial_publisher.cpp

```diff
@@ -0,0 +1,277 @@
+
+#include "ros/ros.h"
+
+#include <iostream>
+#include <cstdio>
+#include "sensor_msgs/Imu.h"
+#include "sensor_msgs/Image.h"
+#include "stereo_msgs/DisparityImage.h"
+#include <camera_info_manager/camera_info_manager.h>
+#include <functional>
+
+// Inludes common necessary includes for development using depthai library
+#include "depthai/depthai.hpp"
+#include <depthai_bridge/BridgePublisher.hpp>
+#include <depthai_bridge/ImuConverter.hpp>
+#include <depthai_bridge/ImageConverter.hpp>
+#include <depthai_bridge/DisparityConverter.hpp>
+
+
+dai::Pipeline createPipeline(bool enableDepth, bool lrcheck, bool extended, bool subpixel, bool rectify, bool depth_aligned, int stereo_fps){
+    dai::Pipeline pipeline;
+
+    auto monoLeft             = pipeline.create<dai::node::MonoCamera>();
+    auto monoRight            = pipeline.create<dai::node::MonoCamera>();
+    auto xoutLeft             = pipeline.create<dai::node::XLinkOut>();
+    auto xoutRight            = pipeline.create<dai::node::XLinkOut>();
+    auto xoutRectifiedLeft    = pipeline.create<dai::node::XLinkOut>();
+    auto xoutRectifiedRight   = pipeline.create<dai::node::XLinkOut>();
+    auto stereo               = pipeline.create<dai::node::StereoDepth>();
+    auto xoutDepth            = pipeline.create<dai::node::XLinkOut>();
+    auto imu                  = pipeline.create<dai::node::IMU>();
+    auto xoutImu              = pipeline.create<dai::node::XLinkOut>();
+
+    // XLinkOut
+    xoutLeft->setStreamName("left");
+    xoutRight->setStreamName("right");
+    xoutRectifiedLeft->setStreamName("rectifiedleft");
+    xoutRectifiedRight->setStreamName("rectifiedright");
```

Comment:
These are not used. You can remove them since you are using xoutLeft and xoutRight. 
Or modify it based on which you are going to use.